contracts/prize-pool/MappedSinglyLinkedList.sol

```diff
@@ -54,4 +59,14 @@ library MappedSinglyLinkedList {
   function contains(Mapping storage self, address addr) internal view returns (bool) {
     return addr != address(0) && self.addressMap[addr] != address(0);
   }
+
+  function clearAll(Mapping storage self) internal {
```

Comment:
I'd like to see tests for this functions as well

Reply:
test added!